common/models/user.json

```diff
@@ -29,8 +29,17 @@
     "emailAuthLinkTTL": {
       "type": "date"
     },
+    "externalId": {
+      "type": "string",
+      "description": "A uuid/v4 used to identify user accounts"
+    },
+    "unsubscribeId": {
+      "type": "string",
+      "description": "An ObjectId used to unsubscribe users from the mailing list(s)"
+    },
     "password": {
-      "type": "string"
+      "type": "string",
+      "description": "No longer used for new accounts"
```

Comment:
This is still going to be used.

Meaning even if we use auth0 via passport or deprecated logins, we are creating a dummy password which is just a random uuid string, that is further hashed same as our older implementations.

Its a LB limitation. And is coming from the parent model.